UnitTests/OIDServiceDiscoveryTests.m

```diff
@@ -152,47 +154,58 @@ + (NSURL *)googleDiscoveryAuthorizationEndpoint {
 
 // from https://accounts.google.com/.well-known/openid-configuration
 static NSString *const kDiscoveryDocument =
-    @"{\"issuer\":\"https://accounts.google.com\",\"authorization_endpoint\":\"https://account"
-      "s.google.com/o/oauth2/v2/auth\",\"token_endpoint\":\"https://www.googleapis.com/oauth2/v4/to"
-      "ken\",\"userinfo_endpoint\":\"https://www.googleapis.com/oauth2/v3/userinfo\",\"revocation_e"
-      "ndpoint\":\"https://accounts.google.com/o/oauth2/revoke\",\"jwks_uri\":\"https://www.googlea"
-      "pis.com/oauth2/v3/certs\",\"response_types_supported\":[\"code\",\"token\",\"id_token\",\"co"
-      "de token\",\"code id_token\",\"token id_token\",\"code token id_token\",\"none\"],\"subject_"
-      "types_supported\":[\"public\"],\"id_token_signing_alg_values_supported\":[\"RS256\"],\"scope"
-      "s_supported\":[\"openid\",\"email\",\"profile\"],\"token_endpoint_auth_methods_supported\":["
-      "\"client_secret_post\",\"client_secret_basic\"],\"claims_supported\":[\"aud\",\"email\",\"em"
-      "ail_verified\",\"exp\",\"family_name\",\"given_name\",\"iat\",\"iss\",\"locale\",\"name\",\""
-      "picture\",\"sub\"]}";
+    @"{\"issuer\":\"https://accounts.google.com\",\"authorization_endpoint\":\"https://"
+    @"accounts.google.com/o/oauth2/v2/auth\",\"device_authorization_endpoint\":\"https://"
```

Comment:
Are there any tests that *don't* include the device_authorization_endpoint? Might be good, since it's optional.

Reply:
I think minimumServiceDiscovery does this https://github.com/soule/AppAuth-iOS/blob/dev-tvos-discovery/UnitTests/OIDServiceDiscoveryTests.m#L91

Reply:
There are some tests that use that i.e testErrorWhenMissingFields

Reply:
I think minimumServiceDiscovery does this https://github.com/soule/AppAuth-iOS/blob/dev-tvos-discovery/UnitTests/OIDServiceDiscoveryTests.m#L91
There are some tests that use that, i.e testErrorWhenMissingFields
(Reposting my reply since it wasn't showing up on conversation page)

Reply:
OK, great!